src/base/docstrings/docstrings.jl

```diff
@@ -0,0 +1,56 @@
+macro _change_bound_s_bang(model_type, index_name, index_type, example, isplural, isinplace)
```

Comment:
can this file go to `reconstruction/modifications/docstrings.jl` ?

Reply:
Unfortunately not, it would break the load order 😞 `reconstruction/modifications/generic.jl` needs the files in `reconstruction` to be loaded before it can be loaded but the opposite is required for `docstrings.jl`...